src/main/Config.cpp

```diff
@@ -147,6 +149,12 @@ Config::Config() : NODE_SEED(SecretKey::random())
     ARTIFICIALLY_SLEEP_MAIN_THREAD_FOR_TESTING =
         std::chrono::microseconds::zero();
 
+#ifdef BUILD_TESTS
```

Comment:
These should just be 0 initialized since the getters return the default value if the TESTING allowances are 0 anyway. Having magic numbers in 2 places, especially behind a compile time flag, is a little confusing.

Reply:
Sounds good, updated (addressed your other comment too)